### PR TITLE
GridDumpingPipeline

### PR DESCRIPTION
**Pull request by mamikonyana**
Creates a new pipeline that will dump scores both into a density grid and before/after score pairs of batchFeatureTransform

main goal of this PR is to be able to dump scored grid of the data

### CODE REVIEW

**Comment by mamikonyana** [comment]
@pbailis @edgan8 could you take a look?

File: src/main/java/macrobase/runtime/standalone/MacroBaseGridCommand.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import io.dropwizard.setup.Bootstrap;
+import macrobase.MacroBase;
+import macrobase.analysis.pipeline.GridDumpingPipeline;
+import macrobase.analysis.result.AnalysisResult;
+import macrobase.conf.MacroBaseConf;
+import net.sourceforge.argparse4j.inf.Namespace;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MacroBaseGridCommand extends ConfiguredCommand<MacroBaseConf> {
```
Comment (pbailis) [inline src/main/java/macrobase/runtime/standalone/MacroBaseGridCommand.java]: This won't be necessary after the command line refactor, but okay for now.

File: src/main/java/macrobase/util/DiagnosticsUtils.java [hunk 2]
```diff
@@ -9,19 +9,14 @@
 
 public class DiagnosticsUtils {
 
-    public static List<Datum> create2DGrid(double[][] boundaries, double delta) {
+    public static List<Datum> createGridFixedIncrement(double[][] boundaries, double delta) {
```
Comment (pbailis) [inline src/main/java/macrobase/util/DiagnosticsUtils.java]: Question: why are we dumping a grid instead of just dumping all scores, then post-processing?

**Comment by pbailis** [comment]
LGTM; some of the code won't be necessary after the command line push. You can either remove after I push, or I can remove after you push.

Before merging, one question: why compute the grid inside MB instead of just dumping scores and computing it later?

File: src/main/java/macrobase/util/DiagnosticsUtils.java (hunk 2 above)
Comment (mamikonyana) [inline src/main/java/macrobase/util/DiagnosticsUtils.java]: I want to actually see the distribution in areas without data, also it makes plotting much easier.

**Comment by mamikonyana** [comment]
I'll rebase off master and remove unnecessary code before merging (maybe later tonight)

**Pull request merged**